src/resources/js/views/multiday-events.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Views Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.multiday_events = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} _   Underscore.js
+ * @param  {PlainObject} obj tribe.events.views.manager
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		calendar: '.tribe-events-calendar-month',
+		multidayEvent: '.tribe-events-calendar-month__event-multiday',
+		hiddenMultidayEvent: '.tribe-events-calendar-month__event-multiday--hidden',
+		multidayEventInner: '.tribe-events-calendar-month__event-multiday-inner',
+		multidayEventInnerFocus: '.tribe-events-calendar-month__event-multiday-inner--focus',
+		multidayEventInnerHover: '.tribe-events-calendar-month__event-multiday-inner--hover',
+	};
+
+	/**
+	 * Find visible multiday event that relates to the hidden multiday event
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $hiddenMultidayEvent jQuery object of hidden multiday event
+	 *
+	 * @return {jQuery} jQuery object of visible multiday event or false if none found
+	 */
+	obj.findVisibleMultidayEvents = function( $hiddenMultidayEvent ) {
+		var $calendar = $hiddenMultidayEvent.closest( obj.selectors.calendar );
+		var eventId = $hiddenMultidayEvent.attr( 'data-id' );
+
+		return $calendar
+			.find( obj.selectors.multidayEvent + '[data-id=' + eventId + ']' )
```

Comment:
any reason why we want to do this rather than do it in one pass? doing this we're getting all multiday events, no matter whether they match the id or not, then running through them again. seems like we could avoid the second pass.

Reply:
Lets keep the initial way, like we talked on the Slack.